servers/republik/modules/crowdfundings/lib/AutoPay.js

```diff
@@ -0,0 +1,220 @@
+const debug = require('debug')('crowdfundings:lib:AutoPay')
+
+const { applyPgInterval: { add: addInterval } } = require('@orbiting/backend-modules-utils')
+
+const createCharge = require('./payments/stripe/createCharge')
+const { getCustomPackages } = require('./User')
+const { getLastEndDate } = require('./utils')
+const createCache = require('./cache')
+
+const suggest = async (membershipId, pgdb) => {
+  // Find membership
+  const membership = await pgdb.public.memberships.findOne({ id: membershipId })
+
+  if (!membership) {
+    return false
+  }
+
+  // Find current periods
+  const membershipPeriods = await pgdb.public.membershipPeriods.find({ membershipId: membership.id })
+
+  // Find pledgeOptions
+  const relatedPledgeOptions = await pgdb.public.pledgeOptions.find(
+    {
+      or: [
+        { membershipId },
+        { pledgeId: membership.pledgeId }
+      ]
+    }
+  )
+
+  if (relatedPledgeOptions.length < 1) {
+    return false
+  }
+
+  // Find latest pledge
+  const pledge = await pgdb.public.pledges.findOne(
+    { id: relatedPledgeOptions.map(po => po.pledgeId), status: 'SUCCESSFUL' },
+    { orderBy: { createdAt: 'DESC' }, limit: 1 }
+  )
+
+  // to get membership-related rewardId
+  const membershipTypes = await pgdb.public.membershipTypes.findAll()
+  const membershipPackageOptions = await pgdb.public.packageOptions.find(
+    { rewardId: membershipTypes.map(mt => mt.rewardId) }
+  )
+
+  const pledgeOptions =
+    (await pgdb.public.pledgeOptions.find({
+      pledgeId: pledge.id,
+      'amount >=': 1
+    }))
+      .map(pledgeOption => ({
+        ...pledgeOption,
+        packageOption: membershipPackageOptions.find(po => po.id === pledgeOption.templateId)
+      }))
+
+  const withOtherRewards = pledgeOptions.filter(po => !po.packageOption).length > 1
```

Comment:
```suggestion
  const withOtherRewards = pledgeOptions.filter(po => !po.packageOption).length > 0
```
Why gt 1? Shouldn't it be gt 0?

Reply:
That was flawed indeed, it should have been greater than 0.

Meanwhile, `withOtherRewards` and `withOtherMemberships ` are not used anymore, hence removing these at once.